client/my-sites/email/email-management/home/email-plan-mailboxes-list.jsx

```diff
@@ -18,6 +19,7 @@ import { getGSuiteSubscriptionStatus, getGmailUrl } from 'calypso/lib/gsuite';
 import { getTitanEmailUrl, useTitanAppsUrlPrefix } from 'calypso/lib/titan';
 import EmailMailboxActionMenu from 'calypso/my-sites/email/email-management/home/email-mailbox-action-menu';
 import EmailMailboxWarnings from 'calypso/my-sites/email/email-management/home/email-mailbox-warnings';
+import EmailTypeIcon from 'calypso/my-sites/email/email-management/home/email-type-icon';
 import { recordEmailAppLaunchEvent } from './utils';
 
 const getListHeaderTextForAccountType = ( accountType, translate ) => {
```

Comment:
```suggestion
const getListHeaderTextForAccountType = ( domain, translate ) => {
	if ( hasEmailForwards( domain ) ) {
```
Optional, but this would allow us to drop the `accountType` prop to `MailboxListHeader`.